server/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexResponse.java

```diff
@@ -56,7 +56,7 @@
 
     public CreateIndexResponse() {}
 
-    protected CreateIndexResponse(boolean acknowledged, boolean shardsAcknowledged, String index) {
+    public CreateIndexResponse(boolean acknowledged, boolean shardsAcknowledged, String index) {
```

Comment:
Probably not necessary. I switched it back and everything looks fine.

Reply:
This was leftover from an experiment, thanks for catching.